Bricks/Chapel-Data-Structures/v0.1.0.toml

```diff
@@ -0,0 +1,5 @@
+[brick]
+name = "Chapel Data Structures"
```

Comment:
This should be the name of the module, i.e. `DataStructures`

Reply:
Done.